configs/hybrik/resnet34_hybrik_mixed.py

```diff
@@ -1,5 +1,7 @@
 _base_ = ['../_base_/default_runtime.py']
 
+# evaluate
+evaluation = dict(interval=1, metric='joint_error')
```

Comment:
Joint_error contains MPJPE and PA-MPJPE.

Reply:
To be more clear, we may consider seperating it into `MPJPE` and `PA-MPJPE`. In this way, it will also be easier for developers to add new metric, such as `PVE` and `accel_error`.